src/ml-search.service.js

```diff
@@ -1135,11 +1135,55 @@
       var self = this;
       
       // TODO: what other conditions
+      // [GJo] This looks a bit peculiar. If you have start and limit in params, but no options, then they get lost?
       if ( !options && params && params.options) {
         options = params;
         params = null;
       }
```

Comment:
This is here to support invoking `getValues` with only a facet name and search options for the combined query (no params). If you have `params.start` and `params.limit`, but no options, nothing get's lost.

However, if you have `params.options`, the params would become the search options in the combined query. Since params.options gets set for you, it wouldn't make sense to pass them. However, it could be a little safer if (`&& !(params.start || params.limit)` was added to the if condition.